static_config.yaml

```diff
@@ -1,6 +1,10 @@
 ---
 # This configuration should only have static settings.
 
+# s3 bucket static prefixes
+edx_s3_processed_prefix: prod-edx/processed/
+edx_s3_rejected_prefix: prod-edx/rejected/
```

Comment:
Are these bucket names plus prefixes?

Reply:
Nope just prefixes on the same ingest bucket